back-end/lms/models/student.py

```diff
@@ -32,7 +32,56 @@ class Student(Person):
     birthdate = models.DateField()
     program = models.ForeignKey(Program, models.DO_NOTHING)
     status = models.ForeignKey(Status, models.DO_NOTHING)
-    photo = models.CharField(max_length=128, blank=True, null=True)
+    photo = models.ImageField(upload_to = '', max_length=128)
```

Comment:
Photos may be used outside of the Student model, placement should be discussed with @gakhromov.

Reply:
I guess that we wil store photos in the `Person` model, so no need to for this thing here